src/libraries/System.Globalization.Extensions/tests/IdnMapping/Data/Unicode_11_0/IdnaTest_11.txt

```diff
@@ -1588,15 +1588,15 @@ xn--210d.-; 𑁅.-; [V3, V5]; xn--210d.-; ; ;  # 𑁅.-
 ්򕸽󠧱。饈≠٤; ්򕸽󠧱.饈≠٤; [B1, B5, B6, P1, V5, V6]; xn--h1c25913jfwov.xn--dib144ler5f; ; ;  # ්.饈≠٤
 ්򕸽󠧱。饈≠٤; ්򕸽󠧱.饈≠٤; [B1, B5, B6, P1, V5, V6]; xn--h1c25913jfwov.xn--dib144ler5f; ; ;  # ්.饈≠٤
 xn--h1c25913jfwov.xn--dib144ler5f; ්򕸽󠧱.饈≠٤; [B1, B5, B6, V5, V6]; xn--h1c25913jfwov.xn--dib144ler5f; ; ;  # ්.饈≠٤
-𞥃ᠠ⁷｡≯邅⬻4; 𞥃ᠠ7.≯邅⬻4; [B1, B2, P1, V6]; xn--7-v4j2826w.xn--4-ogoy01bou3i; ; ;  # 𞥃ᠠ7.≯邅⬻4
```

Comment:
did we comment same cases in previous versions?

Reply:
I did have to comment some cases in Unicode_6. I'll see if this cases are present in other Unicode versions and comment them out if needed.

Reply:
I was expecting 6.0 should work without commenting anything more than what we commented before. that is interesting. 

---
In reply to: [435559684](https://github.com/dotnet/runtime/pull/37281#discussion_r435559684) [](ancestors = 435559684)

Reply:
The reason why I needed to comment some is because we were using Transitional test cases. Now we're using NonTransitional for ICU. So I had to comment 4 NonTransitional test cases that were checking for Bidi error codes, and since we don't request IDN to check for Bidi conformance, those cases don't throw.

Reply:
but we still need to run those on Windows. no?

---
In reply to: [435562512](https://github.com/dotnet/runtime/pull/37281#discussion_r435562512) [](ancestors = 435562512)

Reply:
No, in windows we run `Transitional` test cases. Which are already commented out.

Reply:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Globalization.Extensions/tests/IdnMapping/Data/Unicode_6_0/IdnaTest_6.txt#L108